README.md

```diff
@@ -94,6 +94,27 @@ cargo install --locked --path .
 
 Please ensure ports `4133/tcp` and `3033/tcp` are open on your router and OS firewall.
 
+**[*Beta* - For Debian users]** 
+
+Firstly, install the updated version of LLVM and Clang.
+Download the official install script from here (https://apt.llvm.org/llvm.sh) and then run:
+```
+./llvm.sh
+```
+
+A helper script to install dependencies is available. From the `snarkOS` directory, run:
+```
+./build_debian.sh
+```
+
+Lastly, install `snarkOS`:
+```
+cargo install --locked --path .
+```
+
```

Comment:
```suggestion
```
This is already contained in the script, so no need for it; can you also remove it in the Ubuntu docs right above?